fv/winfv/policy_test.go

```diff
@@ -159,7 +159,7 @@ var _ = Describe("Windows policy test", func() {
 
 			// Assert that it's now reachable.
 			err = kubectlExec(`-t porter -- powershell -Command 'Invoke-WebRequest -UseBasicParsing -SkipCertificateCheck -TimeoutSec 5 https://kubernetes.default.svc.cluster.local'`)
```

Comment:
It may require more than `-SkipCertificateCheck` for Invoke-WebRequest to return good status. I've used this script manually
https://gist.github.com/jmassardo/2e0dd7cce292f16ff8f6945b8b3752b5#file-invoke-webrequest_ignore_ssl-ps1-L2

and still hit error. However, I believe connection is successful. 

```
PS C:\> Invoke-WebRequest -UseBasicParsing -TimeoutSec 1 https://kubernetes.default.svc.cluster.local
Invoke-WebRequest : {
  "kind": "Status",
  "apiVersion": "v1",
  "metadata": {

  },
  "status": "Failure",
  "message": "forbidden: User \"system:anonymous\" cannot get path \"/\"",
  "reason": "Forbidden",
  "details": {

  },
  "code": 403
}
At line:1 char:1
+ Invoke-WebRequest -UseBasicParsing -TimeoutSec 1 https://kubernetes.d ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (System.Net.HttpWebRequest:HttpWebRequest) [Invoke-WebRequest], WebExc
   eption
    + FullyQualifiedErrorId : WebCmdletWebResponseException,Microsoft.PowerShell.Commands.InvokeWebRequestCommand
PS C:\>
```

Reply:
Maybe we should just use `nginx` service (created by infra) for WinFV test.